Adapt documentation to new split binaries from C++ rewrite. (#2289)
mendersoftware/mender-docs: 10.Downloads/docs.md

##### Upgrading Mender after the express installation

> After installing the Mender clients with [get.mender.io](https://get.mender.io),

Reviewer: ```suggestion
After installing the Mender Client with [get.mender.io](https://get.mender.io),
```

> After installing the Mender clients with [get.mender.io](https://get.mender.io),

Reviewer 2: ```suggestion
After installing the Mender client with [get.mender.io](https://get.mender.io),
```

the packages are maintained by the package manager. To upgrade them, simply run

<!--AUTOMATION: ignore -->
```bash
sudo apt-get update
sudo apt-get upgrade
```

!!! If you customize any of the installed files from any of the Mender packages (for example
!!! modifying identity or inventory scripts), then please make sure to also save your work in an
!!! additional place.  Files at paths that match the defaults shipped by the packages will be
!!! overwritten when they are upgraded or re-installed, so you might lose your work if you only
!!! modified the original files.

> To prevent the Mender clients from upgrading when upgrading the rest of the

Reviewer: ```suggestion
To prevent the Mender Client from upgrading when upgrading the rest of the
```

> To prevent the Mender clients from upgrading when upgrading the rest of the

Reviewer 2: ```suggestion
To prevent the Mender client from upgrading when upgrading the rest of the
```

<!-- AUTOMATION: execute=apt-get update -->
<!-- AUTOMATION: execute=DEBIAN_FRONTEND=noninteractive apt-get install -y mender-client -->

>  To prevent the Mender clients from upgrading when upgrading the rest of the

Reviewer: ```suggestion
 To prevent the Mender Client from upgrading when upgrading the rest of the
```

>  To prevent the Mender clients from upgrading when upgrading the rest of the

Reviewer 2: ```suggestion
 To prevent the Mender client from upgrading when upgrading the rest of the
```

system, mark the packages to be held with:

<!--AUTOMATION: ignore -->
```bash
sudo apt-mark hold mender-auth
sudo apt-mark hold mender-update
sudo apt-mark hold mender-client
```

## Mender add-ons

### Requirements

> You need two applications for any add-on to function: the [Mender-auth Client](../02.Overview/15.Taxonomy/docs.md)

Reviewer: ```suggestion
You need two applications for any add-on to function: [`mender-auth`](../02.Overview/15.Taxonomy/docs.md)
```

> You need two applications for any add-on to function: the [Mender-auth Client](../02.Overview/15.Taxonomy/docs.md)

Reviewer 2: ```suggestion
You need two applications for any add-on to function: `mender-auth` one of the components of the Mender client, ```

for the installation in a Yocto Project environment.

To install `mender-connect` using Mender APT repository, follow the instructions

> for [installing Mender clients using the APT

Reviewer: ```suggestion
for [installing the Mender Client using the APT
```

Author: No, this one is correct. Note that it's talking about *all* the mender clients, including mender-connect and mender-configure. Potentially we could have a different name for that bundle of applications, but that is a separate discussion (pre-existing).

### Remote Terminal add-on

> The Remote Terminal does not require any items installed other than the Mender-auth Client

Reviewer: ```suggestion
The Remote Terminal does not require any items installed other than `mender-auth`
```

and Mender Connect.

### File transfer add-on

> The File Transfer does not require any items installed other than the Mender-auth Client

Reviewer: ```suggestion
The File Transfer does not require any items installed other than `mender-auth`
```

and Mender Connect.

### Mender Configure add-on

> Mender offers a configure extension (`mender-configure`) to the Mender-update client

Reviewer: ```suggestion
Mender offers a configure extension (`mender-configure`) to the `mender-update` client
```

installation alternatives.

To install `mender-configure` using Mender APT repository, follow the

> instructions for [installing Mender clients using the APT

Reviewer: ```suggestion
instructions for [installing the Mender Client using the APT
```
